source/_integrations/smart_meter_texas.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: Smart Meter Texas
+description: Instructions on how to integrate Smart Meter Texas into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 0.115
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - "@grahamwetzler"
+ha_domain: smart_meter_texas
+---
+
+The [Smart Meter Texas](https://www.smartmetertexas.com/) integration allows Texas residents to integrate their electric meters into Home Assistant. This integration will create a sensor for each meter that shows the current reading.
+
+On its own this sensor is not particularly useful. However, combined with a [`utility_meter`](https://www.home-assistant.io/integrations/utility_meter/) sensor you can obtain usage over a month, for example.
+
+Any Texas electricity consumer can [register an account](https://www.smartmetertexas.com/register). To register you will need following details which can be found on your electric bill.
+
+1. The name of your retail electric provider
+2. Your meter number
+3. Your ESIID (Electric Service Identifier)
+
+<div class='note'>
+Due to API limits, this sensor refreshes once per hour. The limit is 24 meter readings per calendar day.
+</div>
+
+## Configuration
+
+This integration can be configured from the frontend.
+
+Menu: _Configuration > Integrations_
```

Comment:
And click on the `+` sign and search for `Smart Meter Texas`...?

Reply:
Sorry about that. Hopefully it looks good now?